src/main/java/pages/ContactUsPage.java

```diff
@@ -24,4 +40,50 @@ public Boolean isTitleVisible(){
         return driver.findElement(contactUsPageTitle).isDisplayed();
     }
 
+    public Boolean contactFormIsAvailable(){
+        WebDriverWait wait = new WebDriverWait(driver,timeOut);
+        wait.until(ExpectedConditions.presenceOfElementLocated(contactForm));
+         return driver.findElement(contactForm).isEnabled();
+    }
+
+    public void navigateToContactUsPage(){
+        ValoriHomePage valoriHomePage=new ValoriHomePage(driver);
+        MobileChapterPage mobileChapterPage=new MobileChapterPage(driver);
+        ContactUsPage contactUsPage=new ContactUsPage(driver);
+        valoriHomePage.clickCookieConsent();
+        valoriHomePage.clickExpertisesDropdown();
+        valoriHomePage.clickMobileInDropDOwn();
+        mobileChapterPage.clickContactUsButton();
+
+        //switch the tab
+        ArrayList<String> tabs = new ArrayList<String>(driver.getWindowHandles());
+        driver.switchTo().window(tabs.get(1));
+        contactUsPage.isTitleVisible();
+    }
+
+    public void fillContactUsForm(Map<String,String> userInfo) {
+       WebDriverWait wait = new WebDriverWait(driver,timeOut);
+       wait.until(ExpectedConditions.presenceOfElementLocated(contactForm));
+
+        wait.until(ExpectedConditions.presenceOfElementLocated(firstNameInputBox));
+        driver.findElement(firstNameInputBox).sendKeys(userInfo.get("FirstName"));
+        wait.until(ExpectedConditions.presenceOfElementLocated(lastNameInputBox));
+        driver.findElement(lastNameInputBox).sendKeys(userInfo.get("LastName"));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(companyNameInputBox));
+        driver.findElement(companyNameInputBox).sendKeys(userInfo.get("CompanyName"));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(phoneNumberInputBox));
+        driver.findElement(phoneNumberInputBox).sendKeys(userInfo.get("TelephoneNumber"));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(emailInputBox));
+        driver.findElement(emailInputBox).sendKeys(userInfo.get("EMail"));
+        wait.until(ExpectedConditions.visibilityOfElementLocated(messageInputBox));
```

Comment:
Quite a bit of code duplication with all these waitUntils / findElement > sendKeys. Might be possible to abstract this. But to be honest, I don't think this many waitUntils are necessary and you could do with just the first.